admin/listPurchases/listView.html

```diff
@@ -58,17 +58,20 @@
     {
         cls: "option-all-orders",
         el: document.querySelector(".option-all-orders"),
-        filter: function(x) { return true; }
+        filter: function(x) { return true; },
+        category: "all"
     },
     {
         cls: "option-new-orders",
         el: document.querySelector(".option-new-orders"),
-        filter: function(x) { return x.status == "pending"; }
+        filter: function(x) { return x.status == "pending"; },
+        category: "completed"
```

Comment:
Why is the `category` not the same as the `status`? Confusing to introduce a new filtering concept. Why not filter on status instead to be sure we do the same as what the frontend code used to do?